docs/core/compatibility/extensions/10.0/provider-alias-attribute-moved-namespace.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Breaking change: ProviderAliasAttribute moved to Microsoft.Extensions.Logging.Abstractions"
+description: Learn about the breaking change in .NET 10 where the ProviderAliasAttribute class moved from Microsoft.Extensions.Logging to Microsoft.Extensions.Logging.Abstractions.
+ms.date: 05/16/2024
+---
+
+# ProviderAliasAttribute moved to Microsoft.Extensions.Logging.Abstractions
+
+## Change description
+
+In previous versions of .NET, `ProviderAliasAttribute` was defined in the `Microsoft.Extensions.Logging` assembly. Starting in .NET 10, this attribute has been moved to the `Microsoft.Extensions.Logging.Abstractions` assembly.
+
+## Version introduced
+
+.NET 10 Preview 4
+
+## Type of breaking change
+
+This change is [source-incompatible](../../categories.md#source-incompatible).
+
+## Reason for change
+
+This change allows applications that depend on `Microsoft.Extensions.Logging.Abstractions` and use `ProviderAliasAttribute` to avoid taking a dependency on the full `Microsoft.Extensions.Logging` package.
```

Comment:
```suggestion
This change allows applications that depend on `Microsoft.Extensions.Logging.Abstractions` and use <xref:Microsoft.Extensions.Logging.ProviderAliasAttribute> to avoid taking a dependency on the full `Microsoft.Extensions.Logging` package.
```